### PR TITLE
added event mrmcd02017.json

### PR DESCRIPTION
**Pull request by elidata**

### CODE REVIEW

File: data/events/mrmcd-2017.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "accessibility": "",
+    "cfp_end": "2017-07-19",
+    "code_of_conduct": "https://2017/mrmcd.net/parkrules.html",
```
Comment (szabgab) [inline data/events/mrmcd-2017.json]: this URL does not look correct. Could you fix it please?

**Comment by szabgab** [comment]
Also please remove the relevant line from the missing_events.md file.

**Comment by elidata** [comment]
Oddly it passed the tests.
Sure

**Comment by szabgab** [comment]
That's an opportunity to write a test too that will catch similar issues. :)

**Comment by szabgab** [comment]
I am glad you are making progress, but you should not add more events to the same pull-request. Once the tests of the latest push pass I'll merge. Then you should update your local repository from my public repo, then create a branch for the next event.

**Comment by elidata** [comment]
What would the Git bash command to rebase my clone from your repository?

**Pull request merged**